tests/suite.conf

```diff
@@ -0,0 +1,11 @@
+[General]
+timeout: 0.1
+
+[paths]
+exiv2: ../build/bin/exiv2
+exiv2json: ../build/bin/exiv2json
+
+[variables]
+error 58 message: corrupted image metadata
```

Comment:
When I looked at this it was a bit strange for me to have the possibility of having variables with several words. Would you re-consider to have a restriction for the variable names (so that a variable must contain exactly 1 word) ? In that case I would turn this file in something like:

```
error_57_message: invalid memory allocation request
exiv2_exception_msg: Exiv2 exception in print action for file
```

Reply:
Multi word variables are a feature of Python's ConfigParser module and afaik there is no way how to forbid it in the ConfigParser itself. I could of course implement a check that would reject all variables with whitespaces in them, but to be honest I don't find that too bothering, as the variables either appear before the delimiter `:` or enclosed in curly braces.

I'll keep it in mind and will take a look how hard enforcing this would be.